src/pages/cart-management/CartManagement.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+
+// css
+import './CartManagement.css'
+
+// local components
+import CartManagementCard from '../../components/cart-management/cart-management-card/CartManagementCard'
+import PlaceOrder from '../../components/cart-management/cart-management-RHS/PlaceOrder'
```

Comment:
For components also you can make an index.js file and use barrel exports.

Reply:
since the number of components are more, will fix this in a separate PR, thanks for suggestion